src/rm_topk.c

```diff
@@ -93,7 +93,39 @@ int TopK_Add_Cmd(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     for(int i = 0; i < itemCount; ++i) {
         size_t itemlen;
         const char *item = RedisModule_StringPtrLen(argv[i + 2], &itemlen);
-        char *expelledItem = TopK_Add(topk, item, itemlen);
+        char *expelledItem = TopK_Add(topk, item, itemlen, 1);
+  
+        if(expelledItem == NULL) {
+            RedisModule_ReplyWithNull(ctx);
+        } else {
+            RedisModule_ReplyWithSimpleString(ctx, expelledItem);
+            TOPK_FREE(expelledItem);
+        }
+    }
+    
+    return REDISMODULE_OK;
+}
+
+int TopK_Incrby_Cmd(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
Could probably make this (and other functions) static?

Reply:
Good point!